node-api/src/events.rs

```diff
@@ -1,300 +1,744 @@
-// Copyright 2019-2021 Parity Technologies (UK) Ltd. and Supercomputing Systems AG
-// and Integritee AG.
-// This file is part of subxt.
+// This file was taken from subxt (Parity Technologies (UK))
```

Comment:
Not so simple recopy paste from subxt. Needed to jump between versions quite a bit because subxt now depends heavily on two external crates (`scale-decoder` and `scale-value`), which are both  not `no_std` compatible. 

But the tests and examples run, so I don't think there is too much need to look at every detail in this file. Not for review at least, it's nice code nonetheless.